### ISSUE TITLE
Make encryption optional in the whole app

### ISSUE THREAD

**Issue by Wandelstock** [vector-im/element-android#2511]
**Is your feature request related to a problem? Please describe.**
Because bug #2070 is still an issue for me

**Describe the solution you'd like**
Off switch for encryption and a flag that other users will see when starting a chat.

**Describe alternatives you've considered**
I stopped using Element for direct messages because they fail time after time

**Comment by Powersource** [comment]
Yeah I love e2ee but the ux for in in element has been really bad for me so far.
